### PR TITLE
Upgrade detail comparison

### PR DESCRIPTION
**Pull request by ShlomiBalalis**
## PR pre-checks (self review)

- [ ] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: sdcm/mgmt/cli.py [hunk 1]
```diff
@@ -122,8 +122,10 @@ def history(self):
             cmd = "task history {} -c {}".format(self.id, self.cluster_id)
         res = self.sctool.run(cmd=cmd, is_verify_errorless_result=True)
         if self.sctool.is_v3_cli:
-            _, _, _, *history_table = res
+            history_table = [line for line in res if len(line) > 1]
             # The info command returns some unnecessary values: task_name, cron, retry
+            # The number of the extra info is not set, so I just search the for the lines that its length is larger than
```
Comment (fruch) [inline sdcm/mgmt/cli.py:127]: would help if there was an example here

Comment (ShlomiBalalis) [reply sdcm/mgmt/cli.py:143]: Done

File: mgmt_upgrade_test.py [hunk 2]
```diff
@@ -126,8 +126,10 @@ def test_upgrade(self):  # pylint: disable=too-many-locals,too-many-statements
         manager_tool = get_scylla_manager_tool(manager_node=manager_node)
         # make sure that the cluster is still added to the manager
         manager_tool.get_cluster(cluster_name="cluster_under_test")
-        validate_previous_task_details(task=repair_task, previous_task_details=repair_task_current_details)
-        validate_previous_task_details(task=backup_task, previous_task_details=backup_task_current_details)
+
+        with self.subTest("Checking that the details of tasks that were created before the upgrade remained the same"):
```
[the remark this answers is not in the document]
Comment (ShlomiBalalis) [reply mgmt_upgrade_test.py:130]: Which check isn't in a subtest?

[the remark this answers is not in the document]
Comment (ShlomiBalalis) [reply mgmt_upgrade_test.py]: Done

**Review by fgelcer — APPROVED** [review]
LGTM

**Review by fruch — APPROVED** [review]
LGTM

**Pull request merged**